RealmSwift.podspec

```diff
@@ -0,0 +1,28 @@
+Pod::Spec.new do |s|
```

Comment:
No, podspecs are constrainted in their names to match: `/#{spec.name}.podspec(.json)?/`